x-pack/plugins/security/public/views/management/edit_role/components/edit_role_page.tsx

```diff
@@ -118,12 +118,12 @@ class EditRolePageUI extends Component<Props, State> {
     );
   }
 
-  public getFormTitle = () => {
+  private getFormTitle = () => {
     let titleText;
     const props: HTMLProps<HTMLDivElement> = {
       tabIndex: 0,
     };
-    if (isReservedRole(this.props.role)) {
+    if (isReservedRole(this.props.role) || !isRoleEnabled(this.props.role)) {
```

Comment:
```suggestion
    if (isReservedRole(this.props.role)) {
```

otherwise, we'll display "Viewing role" when the role is disabled, but still allow them to edit it.